templates/plugin/csharp/pluginClass.cs.tmpl

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using Tizen.Flutter.Embedding;
+using Tizen.System;
+
+namespace {{tizenNamespace}}
+{
+    public class {{pluginClass}} : IFlutterPlugin
+    {
+        private MethodChannel _channel;
+
+        public void OnAttachedToEngine(IFlutterPluginBinding binding)
+        {
+            _channel = new MethodChannel("{{projectName}}", StandardMethodCodec.Instance, binding.BinaryMessenger);
+            _channel.SetMethodCallHandler(HandleMethodCall);
+        }
+
+        public void OnDetachedFromEngine()
+        {
+            _channel.SetMethodCallHandler(null);
+            _channel = null;
+        }
+
+        public Task<object> HandleMethodCall(MethodCall call)
+        {
+            if (call.Method == "getPlatformVersion")
+            {
+                if (Information.TryGetValue<string>("http://tizen.org/feature/platform.version", out string version))
+                {
+                    return Task.FromResult<object>(version);
```

Comment:
We don't have to use the `async` keyword because the example is not asynchronous.

Reply:
I thought it would simplify the code.

```suggestion
                    return version;
```

Reply:
Yes, it would be cleaner to use async keyword. Can I ignore the CS1998 warning? 
Actually, I plan to fix and improve the embedding and this template further.

Reply:
> Can I ignore the CS1998 warning?

I can't say which way is better for this reason. It's up to your preference.